.changeset/popular-tomatoes-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/react-form': patch
+---
+
+Support returning dirty nested fields from `getDirtyValues`
```

Comment:
I feel like a clearer message would be something in the lines of "`getDirtyValues` detects dirty nested fields", since returning the dirty fields was already supported. Just semantics, but oh well... What do you think?

Reply:
Nope, totally agree. I'm better with code than words haha